### ISSUE TITLE
Can't copy Persona 5 Royal save file

### ISSUE THREAD

**Issue by goodkarm** [Fr33dan/GPSaveConverter#23]
I'm getting this error when I try to copy my ps4 save file to my gamepass versiona of the game. Can you help?

![image](https://user-images.githubusercontent.com/66935013/200015545-cc07a8b0-f1dd-4834-b077-b8125a43f724.png)

I followed the instructions from one of the reddit comments here below but it still won't work:
![image](https://user-images.githubusercontent.com/66935013/200015632-5d269ba7-21aa-4455-b734-b13c8c6bb638.png)

**Comment by Fr33dan** [comment]
I am sorry for taking so long to reply to this. I must have missed the email and haven't looked at the issue list directly in some time.

I wish the person who wrote this reddit post had provided this information here. They've created the 95% of the working file translation and if they had let me know I could have added it to the database then and it would just work without having to do the manipulation of the file translation yourself.

I've used this information to add P5R to the game library. Update to the latest version and if you have the game installed on Steam the converter should be able to move your files now without configuration (all I did here was open the latest version and select the game):
![image](https://user-images.githubusercontent.com/4670572/207901010-2b74d412-425d-42f2-9d3b-27b0e53d68b9.png)

If for some reason your non-Xbox save location isn't automatically located, select the folder that contains folders named "SAVE01". "SAVE02", etc.
